content/en/docs/components/pipelines/installation/standalone-deployment.md

```diff
@@ -162,17 +162,17 @@ and use it as your working directory.
 ### Deploy on GCP with Cloud SQL and Google Cloud Storage
 
 **Note**: This is recommended for production environments. For more details about customizing your environment
-for GCP, see the [Kubeflow Pipelines GCP manifests](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/env/gcp).
+for GCP, see the [Kubeflow Pipelines GCP manifests](https://github.com/kubeflow/pipelines/tree/18e1db1013337d1e7fdc8aa2f9b04c0b73f4726b/manifests/kustomize/env/gcp).
```

Comment:
What do you think about using `sdk/release-1.8` branch (i.e.: `https://github.com/kubeflow/pipelines/tree/sdk/release-1.8/...`), which is fairly stable at this point? 
It may give a slight edge in keep things up-to-date to a certain point, and make it easier to identify which branch the reference points to.

Reply:
That works for me, particularly because it is so stable. I will update throughout.

Reply:
@chensun, done